pallets/maintenance-mode/Cargo.toml

```diff
@@ -14,6 +14,9 @@ frame-system = { git = "https://github.com/purestake/substrate", branch = "moonb
 parity-scale-codec = { version = "2.2", default-features = false }
 scale-info = { version = "1.0", default-features = false, features = [ "derive" ] }
 sp-runtime = { git = "https://github.com/purestake/substrate", branch = "moonbeam-polkadot-v0.9.12", default-features = false }
+sp-std = { git = "https://github.com/purestake/substrate", branch = "moonbeam-polkadot-v0.9.12", default-features = false }
+
+cumulus-primitives-core = { git = "https://github.com/purestake/cumulus", branch = "moonbeam-polkadot-v0.9.12", default-features = false }
```

Comment:
Is this a big compile (basically does it require the polkadot runtime)? If it does, it would be nice to add a cargo feature called something like `xcm-support` so that chains that use it without xcm don't have to add all that compile time. Although this pallet lives in the moonbeam repo, so maybe that is over-engineering. Up to you.

Reply:
I will give this a shot, definitely

Reply:
It does not import polkadot-runtime, and it does not seem to me that it adds a big compile (mainly it adds polkadot-primitives). I think we can leave it as is for now

Reply:
I end up adding the flag. Let me know what you think!